Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -513,6 +515,31 @@ int CLuaAccountDefs::RemoveAccount(lua_State* luaVM)
     return 1;
 }
 
+int CLuaAccountDefs::SetAccountSerial(lua_State* luaVM)
+{
+    SString strName;
+    SString strSerial;
```

Comment:
SString is no longer used, please use std::string instead

Reply:
adjusted 62ebf06